generated/nidmm/nidmm_service.cpp

```diff
@@ -406,8 +406,9 @@ namespace nidmm_grpc {
           break;
       }
 
-      auto status = library_->CloseExtCal(vi, action);
-      response->set_status(status);
+      auto cleanup_lambda = [&](uint32_t id) { library_->CloseExtCal(vi, action); };
```

Comment:
This technically works, but I think the first parameter to CloseExtCal should be `id` here (the one that gets passed to lambda by session repo. This gets generated to `vi` because of `create_args`. Can we do something there to generate it differently?

Reply:
We can add a new method for service_helpers.py called `create_args_for_custom_close` which will skip the first parameter and then we can prepend it with "id". @reckenro , shall I make that change or does whatever is there now seem good? Because anyways vi and id are going to be same.

Reply:
I have made that change, If owners ask to revert I can revert

Reply:
Yeah, I like the change you made to skip over the first parameter and just use `id` that gets passed to the lambda.